media-video/mplayer/files/add-configure-riscv-support.patch

```diff
@@ -0,0 +1,10 @@
+--- MPlayer-1.4/configure       2018-12-15 16:38:24.000000000 +0100
++++ MPlayer-1.4-mod/configure   2021-06-19 20:41:00.735402324 +0200
+@@ -1761,6 +1761,7 @@
+       nios2) host_arch=nios2 ;;
+       vax) host_arch=vax ;;
+       xtensa*) host_arch=xtensa ;;
++      riscv*) host_arch=generic ;;
+       *) host_arch=UNKNOWN ;;
+   esac
+   }
```

Comment:
I'd suggest split this commit into two
1) add patch to support riscv (including change to ebuild)
2) keyword the package
usually we'd suggest to open a bug for adding patches in order to gain maintainer's review or ACK, but in this case, it's perfect to re-use the same bug id

also we'd suggest to use git format patch (adding commit message info if necessary), but since this patch is rather straightforward, so I'm fine with no commit message.

Reply:
Done. I split it.